frontend/src/components/PumpAction.js

```diff
@@ -1,19 +1,24 @@
 import styled from "styled-components";
 import {useState} from "react";
-import {postMinutes} from "../service/timer-api-service";
+import {runPump, stopPump} from "../service/timer-api-service";
 
 export default function PumpAction(){
 
     const [minutes, setMinutes] = useState("")
 
-    const updateTimer = () => {
-        postMinutes(minutes).then(response => console.log(response))
+    const handleClickRunPumpButton = () => {
```

Comment:
maybe just startPump and stopPump?

Reply:
Do you mean the handleFunction or the runPump ?